lib/jxl/encode.cc

```diff
@@ -606,6 +606,40 @@ JxlEncoderStatus JxlEncoderOptionsSetInteger(JxlEncoderOptions* options,
       options->values.cparams.options.predictor =
           static_cast<jxl::Predictor>(value);
       return JXL_ENC_SUCCESS;
+    case JXL_ENC_OPTION_MODULAR_TREE_PERCENT:
+      if (value < -1 || value > 100) return JXL_ENC_ERROR;
```

Comment:
Values above 100 (above 1.0) might be useful, since this variable also influences other things that don't get clamped.

Reply:
Documented this, and made it unbounded. If there's a reasonable max value you're aware of, we can add that as a bound